src/platforms/rcore_template.c

```diff
@@ -476,15 +476,15 @@ int InitPlatform(void)
     if (platform.device == EGL_NO_DISPLAY)
     {
         TRACELOG(LOG_WARNING, "DISPLAY: Failed to initialize EGL device");
-        return false;
+        return -1;
```

Comment:
Actually, afaik, there is no clear convention defined on the function return values when used for errors detection... I tried to avoid that approach for raylib but I can consider it...